ChangeLog

```diff
@@ -1,3 +1,13 @@
+Changes in squid-5.0.5 (02 Feb 2021):
+
+	- HTTP: Do not send keep-alive or close in HTTP Upgrade requests
```

Comment:
Just a clarifying suggestion that may also improve folks ability to find relevant changes. Feel free to ignore.

```suggestion
	- HTTP: Do not send Connection:keep-alive/close in HTTP Upgrade requests
```